grails-app/services/au/org/ala/bie/ImportService.groovy

```diff
@@ -554,24 +560,60 @@ class ImportService {
      * </ul>
      * For each taxon in each list, update that taxon's SOLR doc with additional fields
      */
-    def importSpeciesLists() throws Exception {
+    def importConservationSpeciesLists() throws Exception {
         def speciesListUrl = grailsApplication.config.speciesList.url
         def speciesListParams = grailsApplication.config.speciesList.params
-        def conservationDefaultSourceField = grailsApplication.config.conservationList.defaultSourceField
-        Map speciesListMap = grailsApplication.config.conservationLists ?: [:]
+        JsonSlurper slurper = new JsonSlurper()
+        def config = slurper.parse(new URL(grailsApplication.config.conservationListsUrl))
```

Comment:
Should this fall back to the default conservation list json file if the property hasn't been set or the url is unreachable?